### PR TITLE
Bump agent version to 1.12.2

### PR DESCRIPTION
**Pull request by aaithal**
### Summary

Bumps agent to version 1.12.2

### Implementation details

None. Just updated VERSION and CHANGELOG

### Testing

- [X] Builds (`make release`)
- [X] Unit tests (`make short-test`) pass
- [X] Integration tests (`make test`) pass
- [X] Functional tests (`make run-functional-tests`) pass

New tests cover the changes: Yes

### Description for the changelog

None.

### Licensing

This contribution is under the terms of the Apache 2.0 License: Yes, AWS employee

### CODE REVIEW

**Comment by aaithal** [comment]
A note on:
- [X] Integration tests (`make test`) pass

Even though they failed in the test run of `amazon-ecs-bot`, they seem to be passing [here](https://github.com/aaithal/amazon-ecs-agent/compare/dev...aaithal:v1.12.2-stage). Also reran integ tests and they're passing now here too.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.12.2
+* Bug - Fixed a bug where agent fills up the disk when querying stats for stopped containers.
```
Comment (samuelkarp) [inline CHANGELOG.md]: I don't think this is accurate.  Would it be bad to repeat the entry from 1.12.1?  I think that's still the best description; even though the fix is different it's still the same effect.

Comment (aaithal) [inline CHANGELOG.md]: done

**Comment by richardpen** [comment]
🚢

**Comment by samuelkarp** [comment]
:+1:

**Pull request merged**